source/common/config/well_known_names.cc

```diff
@@ -102,8 +106,19 @@ TagNameValues::TagNameValues() {
   // http.[<stat_prefix>.]fault.(<downstream_cluster>.)*
   addTokenized(FAULT_DOWNSTREAM_CLUSTER, "http.*.fault.$.**");
 
-  // listener.[<address>.]ssl.cipher.(<cipher>)
-  addRe2(SSL_CIPHER, R"(^listener\..*?\.ssl\.cipher(\.(<CIPHER>))$)");
+  // listener.[<address>.]ssl.ciphers.(<cipher>)
+  addRe2(SSL_CIPHER, R"(^listener\..*?\.ssl\.ciphers(\.(<CIPHER>))$)");
```

Comment:
Note that this one was always there, but it was matching `cipher` instead of `ciphers` so I don't think it ever worked.